scg-system/src/main/java/io/telicent/core/auth/SCG_AuthPolicy.java

```diff
@@ -0,0 +1,279 @@
+package io.telicent.core.auth;
+
+import io.telicent.core.CQRS;
+import io.telicent.jena.abac.fuseki.ServerABAC;
+import io.telicent.jena.graphql.fuseki.SysGraphQL;
+import io.telicent.servlet.auth.jwt.PathExclusion;
+import io.telicent.smart.caches.configuration.auth.policy.Policy;
+import org.apache.commons.lang3.Strings;
+import org.apache.jena.fuseki.Fuseki;
+import org.apache.jena.fuseki.server.DataAccessPoint;
+import org.apache.jena.fuseki.server.Endpoint;
+import org.apache.jena.fuseki.server.Operation;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Helper class for generating authorization policies from a dynamically configured SCG instance
+ */
+public class SCG_AuthPolicy {
+
+    /**
+     * Default roles policy applied to all recognised dataset endpoints
+     */
+    public static final Policy DEFAULT_ROLES = Policy.requireAny("roles", new String[] { "USER", "SUPER_USER" });
```

Comment:
- [ ] Should move a lot of this stuff over to constants, most likely back in Smart Caches Core